### PR TITLE
Add fromLonLat and toLonLat convenience functions

### PR DESCRIPTION
**Pull request by ahocevar**

### CODE REVIEW

File: test/spec/ol/proj/proj.test.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ describe('ol.proj', function() {
   describe('transform from 4326 to 3857 (Alastaira)', function() {
     // http://alastaira.wordpress.com/2011/01/23/the-google-maps-bing-maps-spherical-mercator-projection/
 
-    it('returns expected value', function() {
+    it('returns expected value using ol.proj.tranform', function() {
```
Comment (marcjansen) [inline test/spec/ol/proj/proj.test.js]: typo: transform

File: examples/wms-image-custom-proj.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ var map = new ol.Map({
   target: 'map',
   view: new ol.View({
     projection: projection,
-    center: ol.proj.transform([8.23, 46.86], 'EPSG:4326', projection),
+    center: ol.proj.transform([8.23, 46.86], 'WGS84', projection),
```
Comment (marcjansen) [inline examples/wms-image-custom-proj.js]: 'WGS84' is leftover, right?

**Comment by marcjansen** [comment]
I like it a lot! IMO: please merge.

Thanks @ahocevar

**Comment by probins** [comment]
I think the docs for the functions should specify/clarify that this is WGS84 lonlat

**Pull request merged**